core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/base/parameter/network/AzureNetworkV4Parameters.java

```diff
@@ -45,7 +45,7 @@ public class AzureNetworkV4Parameters extends MappableBase implements JsonEntity
     private String aksPrivateDnsZoneId;
 
     @ApiModelProperty
-    private boolean noOutboundLoadBalancer;
+    private Boolean noOutboundLoadBalancer;
```

Comment:
Hey @biharitomi - This field is new and we are changing it to the wrapper because if the property is null the creation of an outbound load balancer will depend on another property (if it's an existing network or not). If we use a primitive here we have to push the existing network logic to the callers.

Reply:
Okay, but this an API change which is not a good idea in most of the cases.